helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -1,8 +1,12 @@
-# Overrides the upstream nameOverride and fullnameOverride
+# Overrides the name for custom resources in the templates folder
 nameOverride: ""
```

Comment:
Does that mean we need to set it in 2 places?

Reply:
Since we still have custom resources (mostly networkpolicies) we need to set the overrides for both root and `vertical-pod-autoscaler` section because the helpers is interpreted twice : once for the root chart and another one for the subchart (thx @hervenicol for the explanation)

Reply:
alright so maybe we need to document this?

Reply:
If possible yes :) But as the nameOverride should not stay for long if we can add those netpols upstream, maybe a comment in values should be enough :)

Reply:
hmmm... did not read the code yet, but the idea was to have it in vpa/xxx so the helper define can "see" it from the top-level chart as well as the vpa sub-chart.
So have it defined only once.

Reply:
You mean getting rid of the overrides at the root level ?

Reply:
yes.

Reply:
I managed to achieve this thanks to go templating "de derrière les fagots". Tell me if that's ok for you guys @hervenicol @QuentinBisson